shared/macros-bash.jinja

```diff
@@ -260,7 +260,7 @@ do
 
                 # Split concrete_rule into head, perm, and tail sections using hash '#' delimiter
                 rule_head=$(cut -d '#' -f 1 <<< "$concrete_rule")
-                rule_perm=$(cut -d '#' -f 2 <<< "$concrete_rule")
+                # rule_perm=$(cut -d '#' -f 2 <<< "$concrete_rule")  # not needed in the remediation, but that's what is stored there
```

Comment:
I think you can remove the unused `rule_perm` variable and update the comment in line 261.